api/src/context.rs

```diff
@@ -134,6 +134,9 @@ impl Context {
         let data = self
             .db
             .get_transactions(start_version, limit as u64, ledger_version, true)?;
```

Comment:
if the writeset is not optional, you don't need this, right? The txns are part of the `tx_outputs.transactions_and_outputs`